application_base.cpp

```diff
@@ -464,16 +455,16 @@ void application::shutdown() {
       std::rethrow_exception(eptr);
 }
 
-void application::quit() {
+void application_base::quit() {
    my->_is_quiting = true;
-   io_serv.stop();
+   stop_executor_cb();
```

Comment:
This can be `app().get_io_service().stop()`

Reply:
No, same as above, the appbase user may change the executor() and we don't want the executor type hardcoded in appbase, no API from `application` should be used in this file.